edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/factory/ParsePhraseTypeOnlyTest.java

```diff
@@ -38,23 +42,35 @@ public void setUp() throws Exception {
     public void testParsePhraseCharniak() throws Exception {
         ParsePhraseTypeOnly charniak = ParsePhraseTypeOnly.CHARNIAK;
         assertEquals("[VBD]", charniak.getFeatures(predicate).toString());
-        assertEquals("[NP]", charniak.getFeatures(arg1).toString());
-        assertEquals("[PP]", charniak.getFeatures(arg2).toString());
+        List<String> sortedList = new ArrayList<String>();
+        sortedList.add(charniak.getFeatures(arg1).toString());
+        sortedList.add(charniak.getFeatures(arg2).toString());
+        Collections.sort(sortedList);
```

Comment:
It is a set of one item. That's why `Set.toString()` given the surrounding `[` and `]` in the output string.